core/src/sumeragi/mod.rs

```diff
@@ -283,6 +313,7 @@ impl Sumeragi {
             latest_block_view_change_index,
             current_topology,
             wsv,
+            delayed_wsv,
```

Comment:
What do you think about name `finalized_wsv` as it represent `wsv` that will never change?

Reply:
I have made the change.